src/ScriptBuilder/CounterNameAndDescriptionConventions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus.Metrics.PerformanceCounters
+{
+    static class CounterNameAndDescriptionConventions
+    {
+        public static string GetAverageTimerCounterName(this string durationName)
+        {
+            return $"Avg. {durationName}";
+        }
+
+        public static string GetAverageTimerBaseCounterName(this string durationName)
+        {
+            return $"Avg. {durationName}Base";
+        }
+
+        public static string GetAverageTimerDescription(string durationName, string durationDescription)
+        {
+            return $"{durationName.GetAverageTimerCounterName()} - {durationDescription} The value is the average duration in seconds during the sample interval.";
+        }
+
+        public const string ProcessingTime = "Processing Time";
+        public const string CriticalTime = "Critical Time";
+
+        public const string LegacySlaViolationCounterDescription = "SLA Violation Countdown - Seconds until the SLA for this endpoint is breached. The value is in rounded seconds. This is an instantaneous snapshot, not an average over the time interval.";
```

Comment:
I would write out SLA and added the abbreviation at the end: Service Level Agreement (SLA)